packages/commonwealth/.gitignore

```diff
@@ -23,6 +23,8 @@ cache/
 stats.json
 serviceAccountKey.json
 
+../../.rabbitmq/
```

Comment:
Added this because locally it kept wanting to commit, unsure why.

Reply:
Do you have a `.rabbitmq` directory?

Reply:
Yes, gets generated when I build my docker container. I'll take another look here in a few to see how/why it got committed this time but not the others.